dashboard/test/ui/runner.rb

```diff
@@ -609,6 +609,7 @@ def cucumber_arguments_for_browser(browser, options)
   # browser version 0 implies the latest version.
   arguments += skip_tag('@no_older_chrome') if browser['browserName'] == 'chrome' && (browser['version'].to_i != 0 && browser['version'].to_i <= 67)
   arguments += skip_tag('@no_safari') if browser['browserName'] == 'Safari'
+  arguments += skip_tag('@no_safariyosemite') if browser['browserName'] == 'Safari' && browser['version'] == '8.0'
```

Comment:
is the browser version check to distinguish from mobile?  If so, it might be more robust to check `&& !browser['mobile']` or `&& browser['platform']&.starts_with?('OS X')`.

Alternately... I believe the existing `@no_safari` will already do what you want because of the case differences in browsers.json, although I haven't confirmed this.

Reply:
Yeah, I noticed we're using `safari` on mobile and `Safari` on desktop but figured it'd be a bad idea to depend on that. :grin:  I suppose for a target called "SafariYosemite" it makes the most sense to say
```rb
browser['browserName'] == 'Safari' && browser['platform'] == 'OS X 10.10'
```
Pushing that change now.  Good idea!